### DISCUSSION TITLE
2023-04-28 - Unsolicited Advice - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by lukecookssw** [SSWConsulting/SSW.Rules.Content#5454] [Chewing the Fat]
Hi Team,

Let's talk about [this rule](https://www.ssw.com.au/rules/unsolicited-feedback/)

**Comment by kikibianc** [comment]
When someone that accepts all feedback as constructive gives them, they expect the other to accept the feedback as constructive as well, too. That is definitely not the case almost all the time - giving constructive feedback must have a time and place, as per the rule :)

**Comment by sethdaily** [comment]
I found the video well-made, but I was so-so on the content.
The video has 2 contradicting theories – Goldie and Uly’s take which is that feedback should always be calculated and delivered in a polite and appropriate manner, and Adam’s take which is that feedback should always be free flowing and it matters less how the feedback is delivered. Since the video provides both viewpoints it might be an effective way for people to reflect on themselves and how they prefer giving and receiving feedback. However, having both takes in the same video means the viewer does not get left with a clear message. I felt that there were two conflicting lessons to be learned.

In theory, I agree with Adam. The more feedback flows, the more we can all improve.
In practice, this is not how humans work. Many people do not put their ego to the side when receiving criticism, and it affects both the reception of the feedback and the working relationship between them and the other person. This can be counterproductive in the long run.

**Comment by jernejk** [comment]
The video was skillfully crafted and delves into a crucial issue that isn't easily tackled or resolved.

TL;DR: Trust and consideration are vital. Limiting ourselves to thinking "It's only me who needs to adapt" or "It's only them that needs to adapt" is fundamentally flawed. We all need to find a balance. 😊

The extended version:
I believe the responsibility of making perfect decisions shouldn't rest solely on one party, whether it's the extreme ends of the spectrum like "The d*%k" or "The snowflake." The most effective solution usually lies somewhere in the middle.

For example, providing feedback to someone who can't act on it isn't constructive. It's essential to gauge whether certain feedback would be beneficial for them. You wouldn't tell a person with a broken leg to walk faster, just to illustrate. However, at times, people need to hear honest (and tactful) feedback, even if it might be harsh or momentarily hurtful. (There's no perfect way to deliver bad news, but there are countless terrible ways)

On the other hand, having no filter is undoubtedly a poor approach. A few years ago in Brisbane, there was someone who gave genuinely valuable feedback but with little regard for delivery. Many found it off-putting, even though they learned a lot once they set their emotions aside. Importantly, the feedback wasn't rooted in malice or ignorance.

Unsolicited feedback stemming from ignorance, dishonesty, deception, or malice is highly problematic. People who have dealt with such individuals tend to become more guarded, making it understandable why they would hesitate to accept feedback. Being on the receiving end for a long time can lead to the automatic raising of mental shields against any kind of feedback, even when well-intended.

**Comment by 2fernandez** [comment]
![image](https://user-images.githubusercontent.com/27670973/234901185-d20cb2f9-cd61-4700-9f16-4a8c923a52b0.jpeg)

**Comment by calumjs** [comment]
Great topic. Although I disagree with most of the rule it was a nice thought provoking experience. 

I think unsolicited feedback from complete strangers can be more useful than that coming through "appropriate channels". I don't think it's a good idea to self-disqualify just because you don't know the person you are giving feedback to. Sure, it could also be terrible useless feedback, but it's very easy to disregard in this case. 

I also think that keeping any unsolicited feedback to myself if it doesn't benefit me is a bad idea. If you see someone making a mistake, or doing something in a sub optimal way, why not help them out? I would like to think someone else would do that for me. 

I do agree that it's a good idea to not be a dick, to phrase your feedback appropriately, to give constructive feedback, but I think these are just the base rules to better social interactions and should not result in withholding feedback. 

Ultimately I think the rule should be changed from giving unsolicited feedback to receiving unsolicited feedback. Because this is where the "feedback filter" should be. Include tips like - Don't take it personally. Understand you can't please everyone all the time. Know that unsolicited feedback is just someone's opinion (and therefore can be wrong). Pay attention to useful feedback and ignore useless feedback. Enjoy the benefits of a culture of open and honest feedback. Etc

**Comment by leehawkinsssw** [comment]
SSW has an astonishingly strong feedback culture, I suspect driven from the top 🎩(i.e. Adam). I found this confronting when I joined the company and some aspects of this culture still feel like they go too far to me. As an example, running the retro immediately after a presenter finishes up a presentation honestly makes me highly reluctant to ever participate, as I know this is the time I'd be least ready to take on feedback. 

With this in mind, the rule is really important and I thank Matt G for his considerate approach and the worthy suggestions in it.✅

My 3-part reminder when considering giving (especially unsolicited) feedback is the classic "Is it true? Is it necessary? Is it kind?". I've found this reminder valuable in helping me to take pause ⏸️and then often reframing my feedback or simply not sharing it.

**Comment by leahy268** [comment]
Personally I love receiving feedback and I'm happy to give feedback as well.  Unfortunately I know some people don't like feedback and although I could paraphrase Calum and tell them to take 2 teaspoons of cement that isn't really helpful to them.

Therefore on the whole I agree with just remembering to ask those questions to yourself.  Personally I would never comment on what someone was wearing unless it was a compliment as quite frankly I don't care what someone looks like.  I only care how they treat me.  However asking "is my feedback helpful" and "am I the right person to give that feedback" is a very good start.

**Comment by ozdemir-mehmet** [comment]
Giving unsolicited feedback can be risky and potentially harmful to relationships, but if done tactfully and constructively, it can be helpful.

- Unsolicited feedback can be perceived as criticism or control and may harm relationships.
- Feedback given without full understanding of the context and goals may be inaccurate or unhelpful.
- Constructive and supportive feedback, given with sensitivity and tact, can be beneficial.
- Starting with a positive comment or compliment can make the recipient more receptive to feedback.
- Being willing to listen to the recipient's perspective can help build trust and strengthen the relationship.

**Reply by JeanThirion** [reply]
ChatGPT detected

**Comment by bradystroud** [comment]
The envelope a letter comes in doesn't define its content.
Badly delivered feedback should not be disregarded

**Comment by MarcusIrmscherSSW** [comment]
Great Chewing the Fat,

Unsolicited feedback can be both helpful and harmful, depending on how it is delivered. It pays to consider the time and place that you give that feedback, it may go a long way as to how its received.

**Comment by vladislav-kir** [comment]
Despite video being quite lengthy, in my opinion, it is well made and is watched in one breath.

I agree with Goldie's points, that feedback must be delivered carefully for the best results.

**Comment by drwharris** [comment]
We should always strive to communicate well, however we should always strive to find the message in what is being communicated to us.

**Comment by AlexRothwell** [comment]
I agree with a lot of what the rule and video lays out however I think it is very important to check whether the recipient is okay to receive the feedback _now_. If for instance they've been having a bad day, the feedback may not be taken well and just contribute to making their day worse. In this case, it'd be unlikely to be actioned and may impact on your relationship with the person.

That's not to say that you should never give the feedback if the recipient isn't okay for it to be given at the time, just check in again later when they may be in a more receptive mood

**Comment by jakebayliss** [comment]
Do _**I**_ need to say this?
Disagree - I don't think how well you know someone should determine whether or not _**you**_ give them feedback. If a new starter came in and I don't know them well, I will still give them some feedback. I think it is always okay to give as long as it is respectful.

Do I **_need_** to say this?
Disagree - If I was doing something wrong or there was something I could do better I would want to know about it. Again, it should remain respectful.

Do I need to say **_this_**?
Agree - 100% agree with this. It is always going to be the words used that matter most. However, I disagree with the example used, "I don't like your haircut" is not feedback. That is an opinion and telling someone that is just rude.

I think as long as the feedback isn't rude/disrespectful then it should be okay - especially given SSW's feedback culture

**Comment by camillars** [comment]
When it comes to giving feedback, I believe we should be mindful and take care to consider the situation. It's important to keep in mind that the person giving feedback may not always have the full picture or all the details. We need to be careful about this because otherwise, the feedback may end up doing more harm than good. Nobody wants to feel like their skills or knowledge are being questioned, right? So let's try to approach feedback with empathy and understanding. ✌️

**Comment by bahjatalaadelssw** [comment]
I don't mind unsolicited feedback because I've personally tried to keep my ego in check by keeping the door to discussion open. I find it a small price to pay for creating a safe space for discussion.

**Comment by JackDevAU** [comment]
Great video and rule. Unsolicited advice if handled correctly and constructively can be a very positive thing. However, it can also be the opposite. I think even if you disagree with the rule, its still good to think about these questions before providing the feedback .

**Comment by cvionnet** [comment]
Providing feedback, especially when it is negative, can be a double-edged sword depending on how it is delivered and received. That is why this week's CTF is important.

When giving feedback, it is always beneficial to approach it with a smile. This helps create a welcoming environment and increases the likelihood of the feedback being accepted.

It is crucial to offer constructive feedback that is useful and contributes to improvement. For example, telling new parents that their baby is unattractive is feedback, but it is unhelpful and offensive (and it is acting like a 🍆).

I like the method of introducing feedback through an invitation.

**Comment by jaydenalchin** [comment]
Love this topic. _Being open to feedback_ and _providing critique in a constructive way_ are two different but important skills that can be learned through exposure, empathy, and practice. Constructive feedback always comes from someone's desire to improve a thing, and any iterative improvement to a thing has value.

**Comment by AntPolkanov** [comment]
That part is so true "_There is a myth that being honest is morally superior to any other course of action, or that the honesty cancels out any other negative aspects of an interaction_". That's just another reminder to check your motivation before saying anything.

**Comment by tombui99** [comment]
This is a great topic to discuss on, although I partially agree with the rules, I'm not a fan of the idea that feedback should always be free flowing and it does not matters how it is delivered. It is true that the more feedback received, the more we can reflect on ourselves and improve; nevertheless, it should not be neglected that it can affect the reception of the feedback, potentially hurt their feelings and demotivate them.

**Comment by MattParkerDev** [comment]
It may not negatively affect me by withholding feedback but in the spirit on constant improvement I think all feedback should be shared. I think the way you deliver feedback has a massive impact on the way people receive it.

I think this a hard rule to perfect when we want it to be applicable to SSW employees as well as the general public. I would like to think we at SSW hold each other to a higher standard of enforcement of the rules, and in general, open, clear, and non-confrontational communication. For this reason I think in SSW no feedback should be withheld, but that the method of delivery should be carefully considered. Of course there are always things that should never be said - "your makeup looks terrible", but I interpret this rule as more applying to people's performance in their role than personal critique.

**Comment by jackreimers** [comment]
I agree with the rule and think that feedback is crucial to learning and improving, so long as it is delivered as respectfully as possible. Having been in environments where feedback was seldom given it made it hard to develop and learn.

**Comment by christoment** [comment]
Great rule! On top of the great tips there, I also think that feedback should be given thoughtfully - environment where feedback are given every single minute is a recipe for demotivation.

Happy to see more of this kind of rule in the future :)

**Comment by amankumarrr** [comment]
Nice rule! Using 'I', 'need', and 'this' can have a really impactful effect in every context of life because sometimes the onus lies on us on how our feedback is received.

**Comment by Ben0189** [comment]
Somewhat disagree with some content of the rules, but I still think this is a great topic. Feedback is important for self improvement, I have gain a lot by being open to feedback.

**Comment by KahaMason** [comment]
Great video & great topic. Feedback is important but it can be a double edged sword if not delivered correctly.

- Intent, delivery and competence is important when providing feedback otherwise it can be received negatively.
- The focus of providing feedback should be creating a positive improvement for the person / subject. I don't think considering whether it negatively impacts you should be the focal point.
- Who delivers that feedback is important, especially if there is an order of precedence in place.

**Comment by JeanThirion** [comment]
Let's start with the video - AMAZING. I love the casual discussion followed by Adam's thoughts on it. We should use that as a template, it's really awesome.

On the topic: I find it highly interesting. I didn't realised before doing the CTF that this topic is deeply cultural. I expand much more in my replies in the form itself, but basically it boils down to what's appropriate to say and what's not, aka what society accepts or not. And that is obviously extremely different depending on your culture.

Example: I value honest, direct feedback (especially negative) 10x more than metaphores aiming at not harming my feelings. That's because I value Honesty, Courage and Confrontation. Some people will be the exact opposite, and will hate being confronted directly. So yes I agree that you have to be careful with the way to deliver feedback, but it's more about emotional/social intelligence and understanding how to best deliver feedback based on your interlocutor.

**Reply by leehawkinsssw** [reply]
You bring up a very important point about culture. 

I know from personal experience of working with teams in China for many years that a different approach to feedback was required there, to make sure that the recipient didn't lose face amongst their colleagues. Contrast that with the uber-direct approach to communication (and feedback) in teams I worked with in the Czech Republic. 

Some work trips saw me working on-site with my China teams and Czech teams back-to-back, that was some context-switch!

**Reply by sethdaily** [reply]
Interesting. Culture can make for different expectations, and we shouldn't forget it.

**Reply by Hona** [reply]
> Let's start with the video - AMAZING. I love the casual discussion followed by Adam's thoughts on it. We should use that as a template, it's really awesome.

100%, I loved the commentary style from Adam combined with the conversational style of Uly & Matt

**Comment by chrisschultzssw** [comment]
I generally agree with the rule and video - but I think we need to define "unsolicited". If it is within your own team, or with a person you trust, I think feedback is important for progress.

Similarly, if you put yourself out there - for example, posting a video on YouTube, you can expect to get all sorts of feedback. In this, case, there is more onus on the receiver to take the feedback well.

**Comment by KostaMadorsky** [comment]
What an insightful and engaging video, Adam, Matt, and Uly! The topic of feedback is indeed crucial, and the way you've presented the different aspects, including the D-BAD principle, is both informative and easy to understand. I especially appreciate how you've highlighted the importance of retrospectives and the concept of praise in public, criticize in private. You've successfully managed to address the nuances of giving and receiving feedback while keeping the discussion light and enjoyable. Keep up the great work, and I'm looking forward to more content like this from the SSW team!

**Comment by SamBWagner** [comment]
I like this rule for it's thought provoking nature. However, I don't agree with a lot of the content. Criticism is generally speaking a neutral thing. It's delivery might be positive or negative but the criticism itself is neutral. Most people do solicit feedback/criticism implicitly when making an action. If you perform an action which results in others being aware of said action, then you've implicitly made yourself open for criticism, and I would go as so far as to say you've in most cases, solicited it. 

I believe working on parsing criticism through it's packaging and working on the delivery of your own criticism are more important skills to hone. 

I think in this case, like AI, more data == more good ✅

**Comment by Hona** [comment]
After answering the CTF questions I have a few simmered down thoughts:

- Any confrontation should have a positive outcome, if there is no outcome you additionally frame with the criticism, then don't
- Don't be so rude & egotistical that you say anything & take no responsibility for framing & was people interpret things
- On the flipside, take feedback and extract the part that means something, you don't control how someone talks, but you can control how you respond. Always take the high road, and keep acting professionally no matter what.
- "I'm just being honest here" CAN be a valid 'reason'. "I'm being honest here because I want to see you improve". If the outcome is positive, and you need to be honest that is valid.
- In my personal life, comment about my appearance, or anything for that matter, as that is the only way I can improve in those ways. However, I understand being professional and that is not my advice for the workplace. I think there is some middle ground however. Personally, no matter what it is, the embarrassment of something being brought up is 100x better than never knowing and embarrassing yourself every day

**Comment by jontygardner** [comment]
Welcome all feedback. What a great company we have where we get so much feedback! Then from there, you can decide what to take on board and what to discard.

**Comment by stephanfakossw** [comment]
That's a great topic! I enjoy feedback!
Also, I fully agree that "brutally honest" people enjoy the brutality more than the honesty - so the feedback is not about being constructive but about the person giving the feedback feel better.

**Reply by tomek-i** [reply]
**To be honest**, I prefer brutally honesty  :-P

**Reply by jernejk** [reply]
I think Slavic people, in general, are used to brutality. 😂

We got [Brutalism](https://en.wikipedia.org/wiki/Architecture_of_Yugoslavia#Brutalism) from Soviet Union times (which lasted much longer than in other countries), our jokes are often quite brutal if you think about it, and we love to criticize brutally honestly. (not Slovenians, though, they are indirectly brutal with sarcasm, gossip and giving feedback by complaining loudly about it to someone else, while the criticized person is still in earshot 😂😅)

PS: Slavic are very nice and kind people, we just lean toward dry and sarcastic humor due to many reasons. 😊
They are not afraid to rip off the band-aid without a warning if they see you need that.
I'm terrible example of my own people, exceptionally weird even by Slovenian standards. 😂

**Comment by tomek-i** [comment]
I think the following was kind of covered by the video. Sometimes we need to double check if the feedback we are giving is objective (enough) or contains too much of our personal preference.

**Comment by PennyWalker** [comment]
Feedback needs to be constructive, respectful and clearly actionable. Feedback for the sack of it doesn't help anyway. Timing is also important, it's ok to be honest, but you also have to consider if the timing was appropriate. I always think of the Simon Sinek Delayed Honesty video, it's an interesting take on why timing is important! https://www.youtube.com/watch?v=S5h4h166OUE

**Reply by jernejk** [reply]
Liked, subscribed and rang the bell. 😂
Great tip!

I am usually too soon or way too late with the feedback. 😅

**Comment by AttackOnMorty** [comment]
Giving feedback should be a casual conversation where the goal is to help people improve. In addition, feedback should be actionable.

**Comment by zacharykeeping** [comment]
Overall a very good video and rule. Feedback is important but it's best to make sure it's provided in a way that's constructive and respectful.

**Comment by danielmackay** [comment]
DBAD! 😆😆😆

I think this is a great, rule and all are good points to consider before giving feedback.  I particularly like the distinction between free-flowing internal feedback vs measured external feedback, and also the idea of praise in public, but criticizing in private.

The 'Shame Burrito' reminds me of another feedback methodology called the 'Sh*t Sandwich', where you give praise, criticism, then praise.  Terrible idea that some people seem to be fond of.

Asking if someone is open to feedback is really important.  Sometimes the feedback may be great, but the timing may not be.  We never know what might be going personally or mentally with the other person.

I also think we do need to be conscious of how much feedback we feel the need to impart.  It may be more effective to drip feed feedback instead of letting it flow thick and fast.

Additionally, I believe sometimes it's ok to let minor things go.

**Reply by jernejk** [reply]
I agree, 'Sh*t Sandwich' is something I hear more often and do occasionally use a form of it not to overwhelm people, especially when doing lots of PR reviews.

I try to give some positive feedback on PRs when I see something cool and say explicitly what is optional feedback that can be ignored because we shouldn't have a feel of "Ugh, need to fix something again in PR" but instead "Oh, what do they think about that?".

**Comment by christianmorfordwaitessw** [comment]
Definitely a great video and rule. Having feedback that is constructive is crucial and it's also important to give it to someone at the appropriate time.

**Comment by AshAnil** [comment]
The video is well done. I think the rule about unsolicited feedback has a good outcome on what should we think before giving feedback (✅ 3 questions) Feedback should be given and taken in a positive way to improve both personally and professionally.

**Comment by Dhruv-0987** [comment]
The video was very well made with the background music, setting, and content. One thing I found particularly useful was how the rules are structured which makes it really easy to remember and the way they were discussed in the video. Those 3 questions are definitely going to pop in my brain whenever I am about to tell someone "You Look really fat today" hahaha.

**Comment by Aibono1225** [comment]
Very good video and rule. The 3 questions were very helpful in giving feedback. Before giving feedback, make sure it's valuable and constructive.

**Comment by Levijj22** [comment]
Great rule, awesome video!!
Generally speaking, i think all feedback is worthwhile. Just take what you want and disregard the rest.
However i do believe that sometimes its important to ask yourself before you offer advice or feedback: "Am i putting this into practice myself?". Otherwise it may fall on deaf ears.

**Comment by KristenHu** [comment]
Agree with the rule, even when giving feedback, it's definitely better to use gender-inclusive language instead of speaking roughly. This makes it easier for people to accept the feedback and avoid quarrels.

**Comment by jimmidier** [comment]
Nice made video. Imho it's only the second question that matters. One can always choose what to hear with self interest. It only has true difference when not bringing out the feedback at all. Thinking about diplomacy.

**Comment by jimmy1992abc** [comment]
I agree with most of them. It's important to have open channels for communication and to allow individuals to express their opinions in a respectful manner. While there may be differences in the way people express themselves, the focus should be on the substance of the feedback rather than the style. As long as feedback is provided through appropriate channels and in a constructive manner, it can be valuable for growth and improvement.
Art of speech is important but should not be a block to sharing your ideas.

**Comment by landonmaxwell** [comment]
This was a great one!
If you do not have an established relationship with the recipient of the feedback, you should definitely consider if you should be the one to give feedback.
This is because:
1. Your feedback may be lost or disregarded based on how it reaches them.
2. It could offend, frustrate, or anger the recipient, damaging your relationship.

E.g. If feedback is invited openly, then yes, you should give it. 
Otherwise, if you are not an expert in the field, your opinion may be disregarded as it is liable to a number of cognitive biases including:
- availability heuristic - you don't have access to all the details, so can't make a complete analysis
- confirmation bias - you are only focusing on information that confirms your preconceptions
- egocentric bias - you only know what you have seen or experienced personally
- Dunning-Krueger effect - the tendency for unskilled individuals to overestimate their own ability

**Comment by jerryluossw** [comment]
Great rule and video. I strongly agree to the 3 questions.
In addition, I think sometimes "I'm just being honest" is not a self-delusion. When a member hurts the team, you should give him feedback if you are team manager, at that time," I'm just being honest" usually means it's very serious and important feedback, the member should hear and understand it.

**Comment by Marxoz** [comment]
First, I liked the format of the video. It was enjoyable to watch.

As for the rule, I think it is good to for guiding people on giving feedback. There are so many factors to take into account though, like different personalities etc. So it might be worth adding some more examples of wrong and right to each one of the channels.

**Comment by JaneYangssw** [comment]
Great rule! It's necessary to avoid offending others when giving feedback.

**Comment by Berber-Mama** [comment]
Great video except Adam's background has a little mess on the left side.
Matt's rule is very close to what I did, so I 100% agree. 
And I have a tip that might help - when it comes to something out of my responsibility or my area of expertise, and I really want to give feedback - instead of directly disagreeing, I would ask questions. 
e.g. I think Jerry's code sucks
❌ Sorry Jerry but I have to say that your code looks not tidy here. 
✅ Hey Jerry, may I know why you write the code this way but that way? 
 
You could be wrong, and asking questions make you look humble. People don't get angry with humble ones.

**Reply by jimmidier** [reply]
Hey Jerry, may I know why your code sucks?

**Reply by Berber-Mama** [reply]
🤔

**Reply by jerryluossw** [reply]
![1682652885643](https://user-images.githubusercontent.com/68266237/235048249-f498b360-3461-4c90-995f-190b32ae999b.png)

**Comment by AnkyChe** [comment]
Great video, the format made me feel very comfortable.
The rules offers 3 practical ways to decide whether to offer unsocited advice or not. However, it's still hard to decide whether it's approriate and their might be a chance that people choose not to give advice even if it's a good one.

**Comment by sylhuang** [comment]
Can't tell if I'm actually for or against the rules, but I think if something is so important to me that I have to give feedback, there is no appropriate other channel but me myself to address this. If something is not as important to me (if I don't even care if the feedback is taken) or me giving feedback will not result in a change, then there is no need to give this feedback at all.

**Comment by tino-liu** [comment]
Great video!
I've found that asking the right questions in the right context can be an easier way to provide feedback. When we point out someone's mistake in a direct manner, it can feel like a sword pointed at them, making them raise their defenses. Feedback, however, is meant to be a conversation that provides guidance. By approaching it as a conversation, we can ease into giving feedback and help others grow.

**Comment by ozair-ashfaque** [comment]
Feedback is important but delivery of it is more important.

**Comment by johnxu16** [comment]
Good video and insightful content!

It is common that I feel inappropriate to leave advice so in most cases I would save it in my mind.
Could we have a video instructing us how to offend others or make standpoints much more confirmative?

FYI, Is it designed on purpose that Uly and Matt looked blurry at the first scene they showed up?
https://youtu.be/dLgsQvvYqSQ?t=20

**Comment by Abf09** [comment]
The video kept my interest, Good job SSWTV
I consider this as a very important rule, specially it should only be on need to be basis and not intend to offend anyone

**Comment by PothieuG** [comment]
Interesting topic! Thanks for the video :)

As a junior developer, I often struggle to feel legitimate in my feedbacks. Maybe I feel like I don't have enough experience to give useful feedback. But I think it gets better with time.
So I wonder: even if a feedback isn't very useful (due to lack of experience), is it still a good feedback?

**Comment by zl24622** [comment]
Love this week's CTF:
   - Rule is simple and clear.
   - Valuable advices.
   - Video's timing is good length (not too long) and Matt G presented very nicely. 

"The Honesty Myth" is a good insight to me. Agree with all of the 3 points. Not exactly follow them, sometimes they may be broken due to the actual context. But generally they are good guidances for me to consider when providing feedbacks.

**Comment by Freego1783** [comment]
Important topic! 

I think the most important aspect of this is the delivery of the feedback. Doing your best not to offend anyone can go a long way. Also the timing can be crucial. Knowing the right time to deliver feedback can be a strong skill to have.

**Comment by rubycogan** [comment]
A lot of the time, someone can offer useful feedback because of their unique position/perspective. Therefore if this rule was implemented 100%, only people in positions of power may feel comfortable providing feedback when they are the least affected by said situation or don't have the full picture. If several people gave feedback on a situation from their various perspectives, the feedback would be much more effective...

**Comment by matt-goldman** [comment]
It's really interesting reading through these responses. It's particularly interesting to see how the distinction is being blurred between how to _deliver_ feedback and how to _receive_ feedback.

It's absolutely critical to not take things personally, and consider and take on board the feedback offered to you by _anyone_, under any circumstances. But that's not the point of this video and rule.

While I agree that everyone should be able to handle criticism and take feedback, and ignore the delivery (@bradystroud put it really well), the same can be true of driving through traffic lights. The cars perpendicular to me _should_ stop if they have a red light, but that doesn't mean I'm going to drive through without looking.

If the feedback you want to give someone is important _to you_, then it makes sense for you to ensure you deliver it in a way that is going to be effective. It's your responsibility to take feedback well, but you can't control how other people are going to respond to feedback. They _should_ be mature enough to take the feedback objectively and not respond emotionally to it. But that's not how human beings work. Sure, you can absolutely offer unfiltered and free-flowing feedback, and think of it like a UDP packet: once it's out there, it's the recipients responsibility to pick it up. But if it's _important_ to you, if you _care_ about the recipient taking it on board, then putting some effort into the delivery is going to benefit you.

**Reply by calumjs** [reply]
> If the feedback you want to give someone is important to you, then it makes sense for you to ensure you deliver it in a way that is going to be effective.

> But if it's important to you, if you care about the recipient taking it on board, then putting some effort into the delivery is going to benefit you.

Agreed - that's why I disagree with the portions of the rule suggesting that unsolicited feedback is "more often than not, best to keep to yourself".
I think keeping it to yourself would only make sense if it is not important to you (in which case it's less feedback and more opinion/commentary).

**Reply by matt-goldman** [reply]
That's a great point. Perhaps I should update the rule to provide guidance on the difference between opinions and feedback.

**Comment by TobyChurches** [comment]
I personally believe that feedback is always important, But I definitely agree that it needs to be provided in a polite and constructive manner (and in the right environment) 😁

**Comment by JackDuan123** [comment]
I love this week's topic!!!!!!!!

I played a lot of rank games and basketball games when I was a teenager and now!!

I hate to lose a lot, I hate it! So if someone give me some feedbacks on my game I would appreciate a lot, and it did helped me improved a lot! Especially on basketball. With the beliefs of feedback or suggestions, I gave feedback or suggestions to all my teammates, but you know what? Many people just hate it!!!!

Even when I play now, sometimes I played with a teen player and I usually beat their **** out, when I say you need to use your left hand more, some teenagers love my suggestions but some felt humiliated and told me their "main sport" is football, I felt lucky that they are not boxers.

So it all depends on the person, if someone is passionate about what he or she do, he or she will be very happy to recieve advices

Last but not the least!!!
The way you talk really matters!!!! sometimes the way you talk is more important than the contents of the dialogue!!

**Comment by SofieHong** [comment]
Value for this topic: 5/5 🌟

- Better communication skill
- This applies to many scenarios

I totally agree with the keen relationship between feedback and progress. The most powerful catalyst for the relationship between these two is a friendly and mutually beneficial communication style.  

When giving feedback, it's important to ask yourself a few questions beforehand and phrase things correctly. This can greatly enhance the effectiveness and impact of your feedback. <as per rule: https://ssw.com.au/rules/unsolicited-feedback/> 😜

**Comment by BrookJeynes** [comment]
I agree with provided insightful criticism and feedback if it's for the reason of growth and not just to be mean about it
